test/test_add_page.py

```diff
@@ -29,3 +29,19 @@ def test_add_page_duration(tmp_path):
     pdf.add_page()
     pdf.cell(txt="Page 3")
     assert_pdf_equal(pdf, HERE / "add_page_duration.pdf", tmp_path)
+
+
+def test_break_or_add_page(tmp_path):
+    pdf = fpdf.FPDF()
+    pdf.set_auto_page_break(auto=True)
+    pdf._break_or_add_page()
```

Comment:
Private methods should not be called in tests,
only the public methods should be tested.
You can replace this call by `pdf.add_page()`